src/lib/y2iscsi_client/timeout_process.rb

```diff
@@ -0,0 +1,39 @@
+require "yast"
+require "yast2/execute"
+
+Yast.import "Popup"
+
+# yast2 iscsi client specific namespace
+module Y2IscsiClient
+  # Runner class that execute command with given timeout
+  module TimeoutProcess
+    include Yast::I18n
+    extend Yast::I18n
+
+    # @param [Array<String>] command as list of arguments
+    # @param [Integer] timeout in seconds
+    # @return [Array<Boolean, Array<String>] return pair of boolean if command
+    #   succeed and stdout lines without ending newline
+    def self.run(command, timeout: 10)
```

Comment:
sadly `timeout_s` invokes in me timeout_string so that it should be passed as string...if you want units I prefer to have something like `timeout_sec` or even better `timeout_seconds` or maybe just `seconds` is enough as it is run in context of `TimeoutProcess.run(xxx, seconds: 15)`?

Reply:
I see. I like `TimeoutProcess.run(xxx, seconds: 15)`